### PR TITLE
charliecloud: deprecate old versions

### PR DESCRIPTION
**Pull request by j-ogas**

### CODE REVIEW

File: var/spack/repos/builtin/packages/charliecloud/package.py [hunk 1]
```diff
@@ -18,43 +18,46 @@ class Charliecloud(AutotoolsPackage):
...
+
+    # Autoconf.
+    depends_on("m4",       type="build")
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
-    depends_on("libtool", type="build")
+    depends_on("libtool",  type="build")
 
+    # Image manipulation.
     depends_on("python@3.5:", type="run")
```
Comment (reidpr) [inline var/spack/repos/builtin/packages/charliecloud/package.py]: This should be 3.6 I think.

**Review by reidpr — CHANGES_REQUESTED** [review]

**Comment by tldahlgren** [comment]
@spackbot fix style

**Comment by tldahlgren** [comment]
@j-ogas Looks like you need to fix the style and re-push.

**Review by tldahlgren — DISMISSED** [review]
LGTM (once style issues resolved).

**Comment by tldahlgren** [comment]
@spackbot fix style

**Comment by j-ogas** [comment]
> @j-ogas Looks like you need to fix the style and re-push.

Thanks! I totally forgot about the style linter (ugh, lol). I changed the spacing, hopefully this will work.

**Comment by j-ogas** [comment]
@tldahlgren I have fixed the style issues, however, we now have another Spack CI error here. Do you have guidance on how to proceed? Hope all is well.

**Comment by tldahlgren** [comment]
> @tldahlgren I have fixed the style issues, however, we now have another Spack CI error here. Do you have guidance on how to proceed? Hope all is well.

Thanks.  This is a new check organization.  I'll look into it.

**Comment by tldahlgren** [comment]
@spackbot help

**Comment by j-ogas** [comment]
> > @tldahlgren I have fixed the style issues, however, we now have another Spack CI error here. Do you have guidance on how to proceed? Hope all is well.
> 
> Thanks. This is a new check organization. I'll look into it.

Awesome, appreciate it!

**Comment by tldahlgren** [comment]
Failure *may* be intermittent.  Unclear why a couple of required checks are (still) waiting for status to be reported.  Will close and re-open the PR.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**